### PR TITLE
docs: add link to CDK alternative

### PR DESCRIPTION
**Pull request by blimmer**
This project has been really great and inspired my CDK translation of this repo, https://github.com/blimmer/cdk-static-wordpress. If you're open to it, I added an `alternatives` section to the README.

My project fixes at least: #80, #79, #70, #15, #19, #57, #24, #54, #51, #29, #35, #33, #31, #27, #23, #16

### CODE REVIEW

File: .header.md [hunk 1]
```diff
@@ -218,3 +218,11 @@ If the job fails immediately and your site has previously generated a sitemaps.x
 that generates this file and the crawl job can fail fast if it cannot locate it. For all other features and issues
 relating to WP2Static, [raise an issue on their repo](https://github.com/leonstafford/wp2static/issues).
 For any issues relating to this module, [raise an issue against this repo.](https://github.com/TechToSpeech/terraform-aws-serverless-static-wordpress/issues)
+
+## Alternatives
+
+An alternative written for [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/v2/guide/home.html), is
+available at [`blimmer/cdk-static-wordpress`](https://github.com/blimmer/cdk-static-wordpress). Many open issues
+on this repository have been resolved in the other project. Additionally, the almost any arbitrary property on the
```
Comment (blimmer) [inline .header.md]:
```suggestion
on this repository have been resolved in the other project. Additionally, almost any arbitrary property on the
```

File: README.md [hunk 2]
```diff
@@ -220,6 +220,14 @@ that generates this file and the crawl job can fail fast if it cannot locate it.
 relating to WP2Static, [raise an issue on their repo](https://github.com/leonstafford/wp2static/issues).
 For any issues relating to this module, [raise an issue against this repo.](https://github.com/TechToSpeech/terraform-aws-serverless-static-wordpress/issues)
 
+## Alternatives
+
+An alternative written for [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/v2/guide/home.html), is
+available at [`blimmer/cdk-static-wordpress`](https://github.com/blimmer/cdk-static-wordpress). Many open issues
+on this repository have been resolved in the other project. Additionally, the almost any arbitrary property on the
```
Comment (blimmer) [inline README.md]:
```suggestion
on this repository have been resolved in the other project. Additionally, almost any arbitrary property on the
```

**Comment by blimmer** [comment]
I deprecated my alternative for [several reasons](https://github.com/blimmer/cdk-static-wordpress?tab=readme-ov-file#project-status). Closing.

**Comment by petewilcock** [comment]
Hi @blimmer - apologies for this, I don't even recall this PR being submitted at the time. 

As fun as this project was, I just haven't had the time to maintain it as you've noticed. Aside from desirable additional features, it has several weaknesses that you've also found - WP2Static isn't being developed and in itself is quite slow and flakey. An updated solution would need a completely different method for the static export, and AWS has made a terrible decision in deprecating Serverless v1 and no longer has a true 'Serverless' database offering. That could be overcome as well, but it ends up being a lot of effort with little motivation to do it.

There's enough here for inspiration for someone else, and I see there's been several forks, and I'm happy enough with that!

**Comment by blimmer** [comment]
Yep, I ran into those same issues as well!

**Pull request closed without merging**